### DISCUSSION TITLE
sqlalchemy.orm.mapper.validates() behavior change

### DISCUSSION THREAD

**Discussion by mattiaverga** [sqlalchemy/sqlalchemy#9818] [Usage Questions] [answered]
While trying to migrate code from 1.4 to 2.0 I've been stuck on some tests failing.
Now I have seen that `sqlalchemy.orm.mapper.validates()` had a change in the `include_backrefs` default parameter: in 1.4 it was `True`, while in 2.0 it is `False`.

But I don't think that was supposed to happen: also in the docs the method signature reports:
`function sqlalchemy.orm.validates(*names: str, include_removes: bool = False, include_backrefs: bool = False) → Callable[[_Fn], _Fn]`
but in the description it is said:
`include_backrefs – defaults to True;`

So, while I can now easily fix my tests by manually setting the parameter, I think either docs should be updated or the default should be reinstated as before.

**Comment by zzzeek** [comment, marked as answer]
regression, see #9820
